cityguide/source/app/src/main/java/org/kaaproject/kaa/demo/cityguide/ui/ProgressImageView.java

```diff
@@ -106,6 +105,7 @@ public void onLoadingCancelled(String imageUri, View view) {
     }
 
     private void setDefaultImage() {
+        int defaultImage = R.drawable.ic_launcher;
```

Comment:
It must be in some field, because of you can contribute this code more easily. Default image can be different and this is value